scripts/copy_from_pqclean/copy_from_pqclean.yml

```diff
@@ -70,26 +70,32 @@ kems:
       -
         scheme: "512"
         pqclean_scheme: kyber512
+        upstream_location: KYBER
```

Comment:
The name `pqclean_scheme` seems confusing if we are able to pull implementations from `pq-crystals` and `PQClean`. Would it be possible to change `pqclean_scheme` to something like `scheme_name` or  `scheme_identifier`?

Reply:
Well, I'd argue it still _is_ the `pqclean_scheme` name (as well as the upstream scheme name). If the environment variable KYBER is not set, is _is_ copied from pqclean. If I were thinking this to the end, one could argue that the whole `copy_from_pqclean` (script _and_ variable naming) would have to be re-thought as it's now becoming more of a general 'copy-from-upstreams' facility. But that would warrant _lots of_ further changes and make the PR pretty large and unwieldy and pretty hard to review. I'd thus like to keep it as-is for now. If you feel this change _is_ warranted (and particularly we could utilize it more generally to motivate other upstreams to ready their code for automated integration to `liboqs` -- and maybe the same mechanism could be used by `PQClean` to do _its_ imports more automatically), I'd suggest to open an issue to "Rename copy_from_pqclean script and variables". OK for you?

Reply:
Yeah, that sounds good.